contracts/contracts/chains/optimism/Lib_PredeployAddresses.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+/**
+ * @title Lib_PredeployAddresses
+ */
+library Lib_PredeployAddresses {
```

Comment:
Do we expect this to change (often) in the future?

Reply:
I guess the values here will not change anytime soon, but I nevertheless wanted to have meaningful, searchable names instead of simply a long constant. I think this way makes it more clear where the value came from and would make it easier to search for in the future.